pom.xml

```diff
@@ -28,7 +28,7 @@
     <version>3.0.0-beta-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>${project.artifactId}</name>
-    
```

Comment:
Keep indents (including blank lines) consistent with the previous one.
Ref https://shardingsphere.apache.org/community/en/contribute/code-conduct/